adapters/mobilefuse/mobilefuse.go

```diff
@@ -178,20 +181,41 @@ func (adapter *MobileFuseAdapter) getEndpoint(ext *openrtb_ext.ExtImpMobileFuse)
 	return url, nil
 }
 
-func (adapter *MobileFuseAdapter) getValidImps(bidRequest *openrtb2.BidRequest, ext *openrtb_ext.ExtImpMobileFuse) []openrtb2.Imp {
+func (adapter *MobileFuseAdapter) getValidImps(bidRequest *openrtb2.BidRequest, ext *openrtb_ext.ExtImpMobileFuse) ([]openrtb2.Imp, error) {
```

Comment:
This function does not need to defined as pointer receiver on adapter. This function definition could be rw-written as - 
```
func getValidImps(bidRequest *openrtb2.BidRequest, ext *openrtb_ext.ExtImpMobileFuse) ([]openrtb2.Imp, error) {
```

Reply:
Thanks!
I've removed the receiver from that func def and also from a couple of other functions that did not require a receiver.